src/views/compute/DeployVM.vue

```diff
@@ -80,6 +80,25 @@
                         :options="keyboardSelectOptions"
                       ></a-select>
                     </a-form-item>
+                    <a-form-item :label="this.$t('label.vm.boottype')">
+                      <a-select
+                        v-decorator="['boottype']"
+                        @change="fetchBootModes"
+                      >
+                        <a-select-option v-for="bootType in options.bootTypes" :key="bootType.id">
+                          {{ bootType.description }}
+                        </a-select-option>
+                      </a-select>
+                    </a-form-item>
+                    <a-form-item :label="this.$t('label.vm.bootmode')">
+                      <a-select
+                        v-decorator="['bootmode']"
+                      >
+                        <a-select-option v-for="bootMode in options.bootModes" :key="bootMode.id">
+                          {{ bootMode.description }}
+                        </a-select-option>
+                      </a-select>
+                    </a-form-item>
```

Comment:
This needs to be for vmware and kvm only i think

Reply:
@rhtyd Do you think about this? Because on https://github.com/apache/cloudstack/blob/d4b537efa7ed27ef88b735a22493808b61529186/ui/scripts/ui-custom/instanceWizard.js. I don't see any description for show/hidden of boot type & bootmode

Reply:
@utchoang I think that is an oversight or short cut in the new UI. The UI is being phased out and the feature rather new. I think it slipped attention.